datawig/mxnet_output_symbols.py

```diff
@@ -41,7 +41,7 @@ def make_categorical_loss(latents: mx.symbol,
 
     '''
     fully_connected = None
-    if final_fc_hidden_units is None:
```

Comment:
Can it still be none?

Reply:
I checked this and in the imputer code, if it's passed in as None, then it's set to an empty list.

Reply:
Good catch

Reply:
the None default was because of the problems with collections as default args in python - thanks for catching this one!